app/controllers/concerns/application_controller/login_logistics.rb

```diff
@@ -25,9 +25,14 @@ def post_login_housekeeping
     pref_lang = @user_session.user.pref_lang.to_sym
     I18n.locale = configatron.full_locales.include?(pref_lang) ? pref_lang : I18n.default_locale
 
-    # Redirect.
-    best_mission = @user_session.user.best_mission
-    redirect_back_or_default best_mission ? mission_root_path(mission_name: best_mission.compact_name) : basic_root_path
+    # Redirect admin's first login to password reset.
+    if @user_session.user.admin && @user_session.user.login_count <= 1
```

Comment:
Had the same concern - I first tried it with <= 0 and it didn't appear at all. When I cleared the database and re-created the admin user, <= 1 made it appear only on the first login